v2/server/database/queries/users/localLead/index.js

```diff
@@ -3,12 +3,40 @@ const User = require('./../../../models/User');
 
 module.exports.getAllLocalLeads = () => User.find({ role: 'localLead' });
 
-module.exports.addTrainertoGroup = (localLeadId, trainerId) => {
```

Comment:
this function name was used in `controllers/users/editProfile.js`
so the edit profile throws an error when you try to change the official local lead in edit profile
![Screenshot from 2019-09-24 11-18-12](https://user-images.githubusercontent.com/34629478/65494213-0bd1e100-debd-11e9-9e6b-1ee9d182ad08.png)